src/DynamicFormSelect.vue

```diff
@@ -44,8 +40,21 @@
                 }
             })
 
+            const formOptions = computed(() => {
+                if (props.formControl) {
+                    if (!props.formControl.excludeNullOption) {
+                        const selectOption = {id: "", label: props.selectText}
+                        return [selectOption, ...props.formControl.options];
+                    } else {
+                        return props.formControl.options
+                    }
+                }
```

Comment:
Where has this come from? We're explicitly including an empty option now? Has that actually changed the behaviour, or is it just reproducing what was happening before using TreeSelect?

Reply:
this is reproducing what was happening before but since we pass in the options to the `options` prop in the treeselect component it is replicating this logic in a computed value